README.md

```diff
@@ -59,9 +59,12 @@ Route::group(['prefix' => config('backpack.base.route_prefix', 'admin'), 'middle
 });
 ```
 
-7) [optional] Add a menu item for it in resources/views/vendor/backpack/base/inc/sidebar.blade.php or menu.blade.php:
```

Comment:
We still have a file. `menu_items.blade.php`. 

Maybe it's better to reference it than delete the file reference ?

Reply:
hey @pxpm yes I have done this on v6 docs https://github.com/Laravel-Backpack/docs/pull/483

and Intentionally removed it from repos `readme.md` because they don't have a version readme like doc. I'm trying to cover v6 or older versions.
Adding only the v6 menu view path would lead to confusion.

what do you think now?

Reply:
Yeah sounds good to me